integration_test/third_party_apps_data/applications/postgresql/centos_rhel/install

```diff
@@ -1,11 +1,11 @@
 set -e
 
 source /etc/os-release
-case $VERSION_ID in
+case "${VERSION_ID}" in
```

Comment:
this part of the change is an artifact of me trying something different and then not-quite undoing that attempt. But it's good practice to double quote variables in general.